uint8 addr
uint8 regHighAddr
uint8 regLowAddr
uint8 regNum
uint16 data1
uint16 data2
